src/main/java/model/Player.java

```
@@ -0,0 +1,45 @@
+package model;
+
+
+import exception.InvalidMoneyAmountException;
+
+public class Player extends Person{
+    private int wallet;
+    private int bet;
+    private int playerNumber;
+
+    public Player(){
+        wallet = 100;
+    }
+
+    /**
+     * @return the balance of the player's wallet
+     */
+    public int balance(){
+        return this.wallet;
+    }
+
+    /**
+     * withdraws a certain amount of money from the player's wallet
+     */
+    public void withdraw(int amount) throws InvalidMoneyAmountException {
+        if (this.wallet < amount || amount == 0) {
```

Do we reallz want to throw an exception for amount == 0?